polkadot/node/core/backing/src/lib.rs

```diff
@@ -1163,10 +1062,11 @@ async fn construct_per_relay_parent_state<Context>(
 		let core_index = CoreIndex(idx as _);
 
 		if !has_claim_queue {
+			gum::warn!(target: LOG_TARGET, has_claim_queue, "ClaimQueue runtime api not available. Using availability cores.");
```

Comment:
we should remove the code that handles claim queue not being available

Reply:
Fixed. I've also removed the disabling validators fallback.